src/Raven.Server/Documents/ETL/Providers/AI/Extensions/AiExtensions.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.Onnx;
+using Microsoft.SemanticKernel.Embeddings;
+using Raven.Client.Documents.Operations.ETL.AI;
+using Raven.Server.Documents.Indexes.VectorSearch;
+using GoogleAIVersion = Raven.Client.Documents.Operations.ETL.AI.GoogleAIVersion;
+
+#pragma warning disable SKEXP0070
+
+namespace Raven.Server.Documents.ETL.Providers.AI.Extensions;
+
+public static class AiExtensions
+{
+    public static BertOnnxOptions ToBertOnnxOptions(this OnnxSettings settings)
+    {
+        var defaults = new BertOnnxOptions();
```

Comment:
create a static to avoid creating that each time

Reply:
Done